setup.py

```diff
@@ -23,7 +23,8 @@
 setup_requires = ['numpy']
 install_requires = [
     'chainer>=3.2',
-    'Pillow'
+    'Pillow',
+    'scipy'
```

Comment:
Could you not add this to the requirement?
I want to leave `scipy` as an optional dependency like `matplotlib`.

Please edit `README.md` and `environment.yml` instead.